opennms/values.yaml

```diff
@@ -105,10 +105,10 @@ opennms:
     - RRA:AVERAGE:0.5:288:366
     - RRA:MAX:0.5:288:366
     - RRA:MIN:0.5:288:366
-    alecImage:
-      repository: opennms/alec
-      pullPolicy: IfNotPresent
-      tag: v1.1.1
+    alecImage: # If alecImage is specified, the KAR is retrieved from the specified Docker image used as an initContainer, otherwise the latest release in GitHub is retrieved
```

Comment:
To be safe, I think we should set it as `alecImage: {}` when there is no content within it and let the chart knows that it's a Map.

Reply:
+1. Done.